python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/router-bgp.j2

```diff
@@ -654,6 +660,14 @@ ASN Notation: {{ router_bgp.as_notation | arista.avd.default('asplain') }}
 {%         if router_bgp.address_family_evpn.neighbor_default.next_hop_self_received_evpn_routes.inter_domain is arista.avd.defined(true) %}
 | L3 Gateway Inter-domain | True |
 {%         endif %}
+{%         for segment in router_bgp.address_family_evpn.evpn_ethernet_segment | arista.avd.natural_sort('domain') %}
+{%             if segment.identifier is arista.avd.defined %}
+| {{ segment.domain | title }} Domain: Ethernet-Segment Identifier | {{ segment.identifier }} |
+{%             endif %}
+{%             if segment.route_target_import is arista.avd.defined %}
+| {{ segment.domain | title }} Domain: Ethernet-Segment import Route-Target | {{ segment.route_target_import }} |
+{%             endif %}
+{%         endfor %}
```

Comment:
so with the change I had you make (not testing in both places) it seems this is not triggerd anymore. Does it mean that this is interesting only when EVPN DCI Gateway Summary section is rendered?

Reply:
hmmm so maybe we should keep the test you deleted to exercise the log and remove the on enot triggering it. let me do that quickly - apologies for not catching it earlier